### ISSUE TITLE
Where are the cached images?

### ISSUE THREAD

**Issue by jorgemoralespou** [minishift/minishift#1666]
I have enabled my profile to do image-cache, and I look into the profile and can't see any indication of any image being cached. There's a log file indicating that an image cache process happened, as I can see it in the output of the startup, but the log file is empty.

I have started with --openshift-version=3.7.0-rc.0, and I guess this is the problem.

I just created a new profile without specifying --openshift-version to verify it, and it still there's no images being cached.

**Comment by gbraad** [comment]
```
$ minishift profile set testcaching
$ minishift config set image-caching true
$ minishift start
...
-- Exporting of OpenShift images is occuring in background process with pid 8959.   
...
$ ps grep 8959
# no results, exitcode 1
$ ls ~/.minishift/profiles/testcaching/cache/images/
# no results
```

...

**Comment by LalatenduMohanty** [comment]
I could also reproduce the issue. It is unfortunate that we do not have any integration tests for image caching.  Image caching was introduced as the experimental feature we wanted to move to use OCI image format to store the docker images later.

**Comment by gbraad** [comment]
I have asked @hferentschik to also consider adding a test to the Integrations tests for the new PR #1589

**Comment by gbraad** [comment]
Just to be sure, I also verified if the new solution would work, and the answer is no. This is an issue with the current codebase and needs to be fixed: this is a regression.

**Comment by hferentschik** [comment]
Fixed by pull request #1685

**Comment by hferentschik** [comment]
Closing this issue since with #1716 as fix for #1408 this is not an issue anymore
